### DISCUSSION TITLE
Understanding Godot 4: features that need documentation ASAP

### DISCUSSION THREAD

**Discussion by eon-s** [godotengine/godot-proposals#4051] [General Discussions]
Some users are now testing Godot 4 alpha completely blind, there are almost no guides and examples and it makes difficult to report bugs or usability issues.

I have opened this discussion just to list the big features with problems and I encourage other users to pick them up, create new discussions focused on one of these and start gathering information that could help other testers to report problems before beta and to documentation contributors after feature freeze.

The main features with documentation/information problems are:
1. GDScript 2.0 syntax (docs are updated but outdated)
2. Tilemaps/Tilesets (is hard to understand the new workflow without a guide)
3. Navigation (examples on the progress report are outdated, this is needed for 3.5 too)
4. 3D lights and shadows (is impossible to know the correct settings for different effects)
5. 2D Physics (new nodes and usage)
6. 3D Physics (the same as 2D)
7. Scene importing (there is a new import window with many settings)
8. New text server functionality (theres almost nothing about it)

Am I missing something else?

**Comment by Calinou** [comment]
There's already an issue tracking new features to be documented on the godot-docs repository: https://github.com/godotengine/godot-docs/issues/5121

I would recommend continuing the discussion there instead.

**Reply by eon-s** [reply]
Cool, I don't want to bug docs contributors yet, especially without resources of what to write and changing APIs

**Comment by Xrayez** [comment]
GDExtension: godotengine/godot-docs#5618 lists quite a bunch of aspects.

**Comment by eon-s** [comment]
Looks like documentation now improved a lot, there are still issues with old entries but seems that it's being worked on.
